server/src/main/java/org/elasticsearch/action/support/local/LocalClusterStateRequest.java

```diff
@@ -41,12 +41,23 @@ protected LocalClusterStateRequest(TimeValue masterTimeout) {
      */
     @UpdateForV10(owner = UpdateForV10.Owner.DISTRIBUTED_COORDINATION)
     protected LocalClusterStateRequest(StreamInput in) throws IOException {
+        this(in, true);
+    }
+
+    /**
+     * This constructor exists solely for BwC purposes. It should exclusively be used by requests that used to extend
+     * {@link org.elasticsearch.action.support.master.MasterNodeReadRequest} and still need to be able to serialize incoming request.
```

Comment:
I thought that the Enrich action extended `MasterNodeRequest`, not `MasterNodeReadRequest`?